pkg/plugins/runtime/k8s/plugin.go

```diff
@@ -145,6 +145,7 @@ func addMeshReconciler(mgr kube_ctrl.Manager, rt core_runtime.Runtime, converter
 	}
 	defaultsReconciller := &k8s_controllers.MeshDefaultsReconciler{
 		ResourceManager: rt.ResourceManager(),
+		Log:             core.Log.WithValues("controllers").WithName("mesh-defaults"),
```

Comment:
Should there be a value with the `controllers` key?

Reply:
Could you elaborate?

Reply:
I think it's `WithValues(key, value interface{})`

Reply:
```suggestion
		Log:             core.Log.WithName("controllers").WithName("mesh-defaults"),
```

Reply:
it's actually WithName.WithName. Fixed